### ISSUE TITLE
Quick Start: Browser REPL doesn't launch from file:// URL

### ISSUE THREAD

**Issue by pesterhazy** [clojure/clojurescript-site#51]
Reading https://clojurescript.org/guides/quick-start, the "Browser REPL" section does not work for me. If I follow the instructions and open `file:///path/to/index.html` in a browser (chrome, safari or firefox on macos), I get this exception in the browser console:

```
Uncaught Error: URI file:/robots.txt is invalid for field ppu
    at new goog.net.xpc.CrossPageChannel (repl:2457)
    at Function.clojure.browser.net.xpc_connection.cljs$core$IFn$_invoke$arity$0 (repl:2540)
    at clojure.browser.net.xpc_connection (repl:2539)
    at clojure.browser.repl.start_evaluator (repl:3043)
    at repl:3052
    at Object.goog.events.fireListener (repl:2229)
    at goog.events.handleBrowserEvent_ (repl:2232)
    at repl:2221
```

If I start an ad-hoc HTTP server

```
python -m SimpleHTTPServer 9999
```

and open `http://localhost:9999` in the browser, the errors disappear and the REPL connects. The issue occurs with cljs-1.9.293.jar and cljs-1.9.473.jar.

I don't know where `robots.txt` comes into the picture. Does Google Closure's method of creating an iframe work from `file://` URLs?

**Comment by skeuomorf** [comment]
Same issue.

**Comment by swannodette** [comment]
@pesterhazy the QuickStart doesn't say to use the file protocol. It says to goto a URL in your browser. Browser REPL has a web server built in. If you point at where this confusion comes from I'd like to take a look and get that remedied.

**Comment by pesterhazy** [comment]
@swannodette, reading the guide again now, it clearly says "Point your web browser at http://localhost:9000."

I think when I followed the guide, I opened index.html as a `file` url in the earlier section, where no web server is available and file:// works. Then I reloaded for the Repl section, which failed with a somewhat cryptic error. Though it's true I didn't properly follow the steps, a brief note that index.html needs to be opened via http might help future learners (I'm honestly still not sure why http is needed).
